lazer/contracts/solana/programs/pyth-lazer-solana-contract/src/lib.rs

```diff
@@ -49,15 +52,72 @@ impl Storage {
         &self.trusted_signers[0..usize::from(self.num_trusted_signers)]
     }
 }
+#[account]
+pub struct StorageV2 {
+    pub top_authority: Pubkey,
+    pub treasury: Pubkey,
+    pub single_update_fee_in_lamports: u64,
+    pub num_trusted_signers: u8,
+    pub trusted_signers: [TrustedSignerInfo; MAX_NUM_TRUSTED_SIGNERS],
+}
+
+impl StorageV2 {
+    const SERIALIZED_LEN: usize = PUBKEY_BYTES
```

Comment:
We can make also have some extra space in this account to keep room for adding more stuff in the future.

Reply:
due to the way it's serialized i think extension is reserved for increasing number of signers.

Reply:
if we are going to have upgrades that add fields like this we need to have a way for it so an upgrade doesn't break the config.

Reply:
we can add extra space in between too `_extra_space: [u8; 100]` to reserver 100 bytes.
Also if we think we might increase the trusted signers we should allocate space for it too. The array length can be more than `MAX_NUM_TRUSTED_SIGNERS`.

Reply:
Added.